build-caching/src/main/java/com/vertispan/j2cl/build/TaskScheduler.java

```diff
@@ -294,7 +304,6 @@ private boolean executeFinalTask(CollectedTaskInputs taskDetails, DiskCache.Cach
                             throw new AssertionError("final task marker should have been " + cacheResult.outputDir() + ", instead was " + previous);
                         }
                     }
-                    buildLog.info("--- Ready for browser refresh");
```

Comment:
Having a clear signal on the terminal when the relevant step has been done turned out to be really helpful. Every tasks prints "Finished" something so one needs to read carefully.